lib/volcanic/location/config.rb

```diff
@@ -21,6 +22,10 @@ def domain_url
           @domain_url
         end
 
+        def enable_custom_auth
+          @enable_custom_auth == true
```

Comment:
can we clarify the purpose of this?

it looks like it's defined as a variable but acts more as a check, maybe a question mark might be better? `enable_custom_auth?`

Reply:
yes, is more of a placeholder to check if that setting is enabled